bin/start-collectd

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+: ${GRAPHITE_HOST:=}
+: ${GRAPHITE_PORT:="2003"}
+
+set -eu
+
+CONF_TEMPLATE=/etc/collectd/collectd.conf.tpl
+CONF_DEST=/etc/collectd/collectd.conf
+
+HOSTNAME=$(wget -O - --tries 1 --timeout 0.2 http://169.254.169.254/1.0/meta-data/instance-id 2>/dev/null || hostname)
```

Comment:
Oh, you're right! And I never saw it failing because, well the metadata endpoint isn't available in my local testing. Also seems that we can't go under 1 second timeout, seems like the busybox version of `wget` doesn't support fractions.